src/api.ts

```diff
@@ -399,6 +416,9 @@ class RippleAPI extends EventEmitter {
   computeLedgerHash = computeLedgerHash // @deprecated Invoke from top-level package instead
   signPaymentChannelClaim = signPaymentChannelClaim // @deprecated Invoke from top-level package instead
   verifyPaymentChannelClaim = verifyPaymentChannelClaim // @deprecated Invoke from top-level package instead
+
+  generateFaucetWallet = generateFaucetWallet
```

Comment:
If we're using `axios` to make a HTTP request to the faucet, IMO it doesn't make sense to include this in the `RippleAPI` class. 

Could we add this function to the utils we are exporting?